frontend/src/components/Header/__tests__/header.spec.tsx

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import { render, fireEvent, wait } from "test-utils";
+import Header from "../Header";
+import HeaderLogo from "../HeaderLogo";
+import HeaderNavLink from "../HeaderNavLink";
+import HeaderNav from "../HeaderNav";
+
+describe("HeaderLogo component", () => {
+  it("matches snapshot @snapshot", () => {
+    const { asFragment } = render(<HeaderLogo text="test" />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+  it("includes link when text is passed", () => {
+    const { getByText } = render(<HeaderLogo text="test" />);
+    expect(getByText("test")).toBeInTheDocument();
+  });
+
+  it("excludes link when text is not passed", () => {
+    const { queryByText } = render(<HeaderLogo />);
+    expect(queryByText("test")).not.toBeInTheDocument();
+  });
+});
+
+describe("HeaderNavComponent", () => {
+  let props = {
+    isOpen: true,
+    onClose: jest.fn()
+  };
+
+  afterEach(() => {
+    props.onClose.mockReset();
+  });
+
+  it("matches snapshot @snapshot", () => {
+    const { asFragment } = render(<HeaderNav {...props} />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it("adds visible class when open", () => {
+    const { container } = render(<HeaderNav {...props} />);
```

Comment:
Should this have `isOpen` or `isOpen={true}` and not reply on the default?

Reply:
Agreed, makes it more clear what the test is doing. Fixed.